admin/make_formation.py

```diff
@@ -77,34 +77,31 @@ def generate_t():
 		else:
 			# line member (36 deeper)
 			print(0, end=' ')
-			print_y(8 + ((i - 3) * 36), i)
+			print_y(48 + ((i - 3) * 36), i)
 	print()
 
 # A 'gathered' formation around a centre point.
 def generate_gather():
 	print("GATHER", end=' ')
 	for i in range(num_coords):
-		if i % 3 == 0:
-			if i < 3: # front row
-				xloc = 0 # centre
-				yloc = -36 # front
-			else:
-				xloc = 48 # right
-				yloc = 24 * i // 3
+		if i == 0:
+			xloc = 0 # centre
+			yloc = -36 # front
+		elif i == 1:
+			xloc = 48 # right
+			yloc = -24
+		elif i == 2:
+			xloc = -48 # left
+			yloc = -24
```

Comment:
this block is redundant

Reply:
All 1-3 are at negative y, and all the rest are at positive.